modules/intro/pages/index.adoc

```diff
@@ -10,132 +10,61 @@ The GSQL Language Reference is divided into two major parts:
 * xref:ddl-and-loading:system-and-language-basics.adoc[Data Definition and Loading (DDL) Language] describes system basics, defining a graph schema, and loading data.
 * xref:querying:index.adoc[GSQL Query Language] describes querying.
 
-== Documentation notation
+== What sets GSQL apart
 
-This guide uses conventional notation for software documentation.
-In particular, note the following:
+GSQL is the choice for fast and scalable graph operations and analytics.
+GSQL’s similarity to SQL, high-level syntax, Turing completeness, and built-in parallelism brings faster performance, faster development and the ability to describe any algorithm.
 
-=== Long lines
+=== Designed for analytics
+GSQL is designed for complex graph analytics.
+By taking advantage of a unique feature of the GSQL query language - accumulators, users can leverage TigerGraph's Massively Parallel Processing (MPP) capability without the pain and fuss traditionally associated with parallel programming.
 
-For more convenient display, long statements in this guide may sometimes be displayed on multiple lines.
-This is for display purposes only; the actual code must be entered as a single line (unless the multi-line mode is used).
-When necessary, the examples may show a shell prompt before the start of a statement, to clearly mark where each statement begins.
 
-Example: A `SELECT` query is grammatically a single statement, so GSQL requires that it be entered as a single line.
+=== Turing-complete
+GSQL is Turing-complete with full support for imperative and procedural programming, ideal for algorithmic computation.
+The Turing completeness of the language, especially with the conventional xref:querying:control-flow-statements.adoc[control flow statements], allow users to describe any algorithm with GSQL.
```

Comment:
```suggestion
The Turing-completeness of the language, especially with the conventional xref:querying:control-flow-statements.adoc[control flow statements], allows users to describe any algorithm with GSQL.
```

Reply:
https://en.wikipedia.org/wiki/Turing_completeness#:~:text=In%20colloquial%20usage%2C%20the%20terms,purpose%20computer%20or%20computer%20language.

Wikipedia doesn't seem to use the hyphen